shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -177,6 +177,11 @@ export default {
       return this.$store.getters['management/all'](MANAGEMENT.CLUSTER);
     },
 
+    showClusterTools() {
```

Comment:
These permissions need also to be applied to the tools virtual type in the explorer config.  Looks like `ifHaveType` can be a regex string, so multiple types should work. 

Notice we have `ifHaveType:          new RegExp(`${ MANAGEMENT.SETTING }|${ MANAGEMENT.FEATURE }`, 'i'),` elsewhere... which i don't think should actually work given below? Maybe `RegExp` ctor has smarts to accept regex objs?
- `const haveIds = knownTypes[module].filter((t) => t.match(stringToRegex(p.ifHaveType)) );`?
- ```
    function stringToRegex(str) {
      let out = regexCache[str];
    
      if ( !out ) {
        out = new RegExp(str);
        regexCache[str] = out;
      }
    
      return out;
     }
   ```

Reply:
Done